web-admin/src/routes/[organization]/[project]/-/reports/[report]/export/+page.svelte

```diff
@@ -5,15 +5,15 @@
   import CtaContentContainer from "@rilldata/web-common/components/calls-to-action/CTAContentContainer.svelte";
```

Comment:
@AdityaHegde As per Alex's comment
1. Remove Rill logo for external report users
2. Remove Go To Report Page button for external report users

Reply:
May be we still keep Rill logo, not sure.